extensions/wikia/ContentReview/scripts/contentReviewSpecialPage.js

```diff
@@ -23,14 +23,16 @@ define(
         }
 
         function updateReviewStatus() {
-            var self = $(this);
-            var data = {
-                wikiId: self.attr("data-wiki-id"),
-                pageId: self.attr("data-page-id"),
-                status: self.attr("data-status")
-            };
+            var self = $(this),
```

Comment:
I also think the variable could be something other than `self` which is usually used as an alias for `this` when you need to deal with scope. In this case something like `$button` or something related to what the jQuery element you're storing in it would probably be better.